src/devices/Gpio/Drivers/OrangePiLite2Driver.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
You will need to do 2 line copyright headers like i.e.:

https://github.com/dotnet/iot/blob/master/src/devices/Ak8963/Ak8963I2c.cs#L1-L2